eventsourcing/src/main/java/org/axonframework/eventsourcing/eventstore/StartPosition.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2010-2025. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.eventsourcing.eventstore;
+
+final class StartPosition implements Position {
+    @Override
+    public Position min(Position other) {
```

Comment:
Intellij would complain because you did not add the `@Nonnull` annotation that are present on the implemented interface.